sdk/core/azure-core-amqp/src/test/java/com/azure/core/amqp/implementation/ActiveClientTokenManagerTest.java

```diff
@@ -49,7 +49,7 @@ static void afterAll() {
 
     @BeforeEach
     void setup() {
-        MockitoAnnotations.initMocks(this);
+        MockitoAnnotations.openMocks(this);
```

Comment:
`openMocks` returns a Closeable, we may want to capture that and close it in `AfterEach`

Reply:
Thanks for that catch, I'll do that!